src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -312,11 +312,11 @@ const styles = StyleSheet.create({
   },
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `backgroundColor` property in both `fragmentContainer` and `noDataFoundContainer` styles from `COLORS.white` to `COLORS.almostBlack` are significant. Ensure that this color change aligns with the overall theme of your application and provides a good user experience in terms of readability and contrast.

```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->